src/mozanalysis/experiment.py

```diff
@@ -531,6 +532,43 @@ def _build_enrollments_query_fenix_baseline(self, time_limits):
             dataset=self.app_id or "org_mozilla_firefox",
         )
 
+    def _build_enrollments_query_glean_baseline(self, time_limits, dataset):
+        """Return SQL to query enrollments for a Glean no-event experiment
+
+        If enrollment events are available for this experiment, then you
+        can take a better approach than this method. But in the absence
+        of enrollment events (e.g. in a Mako-based experiment, which
+        does not send enrollment events), you need to fall back to using
+        ``ping_info.experiments`` to get a list of who is in what branch
+        and when they enrolled.
+        """
+        # Try to ignore users who enrolled early - but only consider a
+        # 7 day window
```

Comment:
I don't know why we do this for the fallback mode. In any case, we don't have to worry about it here, since the SDK will only send one enrollment event per client device.